### PR TITLE
Loading and Displaying examples complete 

### PR DESCRIPTION
**Pull request by chst1**

### CODE REVIEW

File: src/image/loading_displaying.js [hunk 1]
```diff
@@ -300,6 +303,46 @@ define(function (require) {
...
+   * @example
+   * 
+   * <div>
+   * <code>
+   * var img;
+   * function preload() {
+   *   img = loadImage("assets/bricks.jpg");
+   * }
+   * function setup() {
+   *   imageMode(CORNER)
```
Comment (kadamwhite) [inline src/image/loading_displaying.js]: should have a semicolon

File: src/image/loading_displaying.js [hunk 2]
```diff
@@ -300,6 +303,46 @@ define(function (require) {
...
+   * </div>
+   *
+   * <div>
+   * <code>
+   * var img;
+   * function preload() {
+   *   img = loadImage("assets/bricks.jpg");
+   * }
+   * function setup() {
+   *   imageMode(CENTER)
```
Comment (kadamwhite) [inline src/image/loading_displaying.js]: ditto -- all `imageMode` lines in the examples are missing `;`

File: src/math/trigonometry.js [hunk 3]
```diff
@@ -24,6 +24,27 @@ define(function (require) {
...
+   * println(a + " : " + c + " : " +  ac);
+   * </code>
+   * </div>
+   *
+   * <div class= “norender">
+   * <code>
+   * var a = PI + PI/4.0;
+   * var c = cos(a);
+   * var ac = acos(c);
+   * //Prints: "3.926991 : -0.70710665 : 2.3561943"
```
Comment (kadamwhite) [inline src/math/trigonometry.js]: Total nitpick: space b/w `//` and `Prints` for legibility

File: src/math/trigonometry.js [hunk 4]
```diff
@@ -79,6 +145,19 @@ define(function (require) {
...
+   * @example
+   * <div>
+   * <code>
+   * function draw() {
+   * background(204);
+   * translate(width/2, height/2);
+   * var a = atan2(mouseY-height/2, mouseX-width/2);
+   * rotate(a);
+   * rect(-30, -5, 60, 10);
+   * }
```
Comment (kadamwhite) [inline src/math/trigonometry.js]: Contents of the `draw() {}` function block should be indented

File: src/math/trigonometry.js (hunk 3 above)
Comment (lmccart) [inline src/math/trigonometry.js]: thanks! i'm pulling this in right now so i'll make the change.

File: src/math/trigonometry.js [hunk 5]
```diff
@@ -95,6 +174,19 @@ define(function (require) {
...
+   * 
+   * @example
+   * <div>
+   * <code>
+   * var a = 0.0;
+   * var inc = TWO_PI/25.0;
+   * for (var i = 0; i < 25; i++) { 
+   * line(i*4, 50, i*4, 50+cos(a)*40.0);
+   * a = a + inc;
+   * }
```
Comment (kadamwhite) [inline src/math/trigonometry.js]: Indent code within the `for() {}` block

File: src/math/trigonometry.js [hunk 6]
```diff
@@ -111,6 +203,18 @@ define(function (require) {
...
+   *
+   * @example
+   * <div>
+   * <code>
+   * var a = 0.0;
+   * var inc = TWO_PI/25.0;
+   * for (var i = 0; i < 25; i++) { 
+   * line(i*4, 50, i*4, 50+sin(a)*40.0);
+   * a = a + inc;
+   * }
```
Comment (kadamwhite) [inline src/math/trigonometry.js]: Indentation

File: src/math/trigonometry.js [hunk 7]
```diff
@@ -127,6 +231,19 @@ define(function (require) {
...
+   * 
+   * @example
+   * <div>
+   * <code>
+   *   var a = 0.0;
+   *   var inc = TWO_PI/50.0;
+   *   for (var i = 0; i < 100; i = i+2) {
+   *   line(i, 50, i, 50+tan(a)*2.0);
+   *   a = a + inc;
+   *   }
```
Comment (kadamwhite) [inline src/math/trigonometry.js]: Indentation

File: src/math/trigonometry.js (hunk 4 above)
Comment (lmccart) [inline src/math/trigonometry.js]: oops, i didn't see the rest of these comments. i'm going to sit down with @chst1 in a moment and we'll do it together.

File: src/math/trigonometry.js (hunk 7 above)
Comment (lmccart) [inline src/math/trigonometry.js]: thanks for the code review! should be all good now.

Comment (chst1) [inline src/math/trigonometry.js]: Thanks for all the help! wont happen again :)

**Pull request merged**